src/UI/webWallet/WebWalletLoginButton/index.tsx

```diff
@@ -28,10 +30,15 @@ export const WebWalletLoginButton: (
     token
   });
   const classes = getGeneratedClasses(className, shouldRenderDefaultCss, {
-    wrapper: `btn btn-primary px-sm-4 m-1 mx-sm-3 ${
-      buttonClassName != null ? buttonClassName : ''
-    }`,
-    loginText: 'text-left'
+    wrapper: `${classNames(
```

Comment:
wrapper reuse

Reply:
I got the idea, but the components from the UI are individual components. Should not share code between them even if there is some duplicate code.
What do you think ?

Reply:
I think we can reuse stuff that is standardized, so when we change this, we can change everywhere at once

Reply:
agree